src/html5sortable.ts

```diff
@@ -145,6 +145,9 @@ const destroySortable = function (sortableElement) {
   const opts = data(sortableElement, 'opts') || {}
   const items = filter(sortableElement.children, opts.items)
   const handles = getHandles(items, opts.handle)
+  // disable adding hover class
+  console.log(opts)
```

Comment:
You forgot a console.log here, can you please delete it, thanks.

Reply:
Oops. My bad 😄